src/som/primitives/actors/PromisePrims.java

```diff
@@ -51,23 +60,46 @@ public boolean matches(final Object[] args, final ExpressionNode[] argNodes) {
              specializer = IsActorModule.class, noWrapper = true)
   public abstract static class CreatePromisePairPrim extends UnaryExpressionNode {
 
+    protected final AbstractBreakpointNode promiseResolverBreakpoint;
```

Comment:
Why in the abstract nodes that are in the SendNode class there is not @Child annotation and in that case are final?